blog/2023-09-20-magazine-005.md

```diff
@@ -0,0 +1,117 @@
+
+![addfile_bucket](./assets/2023-09-20-magazine-005/addfile_bucket_1.png)
+
+### Wing Libraries
+❤️ Added by [Chris Rybicki](https://github.com/Chriscbr) ❤️
+
+With the new subcommand 'wing pack' you can now package up your wing project as a library. This feature enables users to create an npm-compatible tarball that can be published and installed via npm. Even though the tarballs contain Wing files and are not strictly JSII libraries, they can be consumed in Wing in a similar fashion to other JSII libraries.
+
+### Directory Modules
+
+Another addition from Chris is the ability to bring in and access entire directories instead of just individual files. Now, within these directories, users can access types, subdirectories, and their contents. This enhancement not only aids in organizing projects within Winglang but also serves as a foundational step towards supporting Wing libraries.
+
+
+### `fs` Namespace
+❤️ Added by [Jade Zheng](https://github.com/jianzs) ❤️
+
+Wing now has a comprehensive file system namespace. This namespace not only covers standard file operations but also integrates unique functions for reading and writing JSON and YAML. An awesome feature is the normalization of file paths: by default, all returned paths from these operations will use forward slashes, standardizing file paths across platforms. We believe this approach will provide consistency and ease for developers working in mixed platform environments.
+
+
+### `wing test -t tf-aws` Now Runs During CI
+❤️ Added by [Tsuf Cohen](https://github.com/tsuf239) ❤️
+
+Wing's testing system has evolved. Initially, tests ran in a simulator, but to ensure local functionalities work seamlessly in the cloud, tests now also run in the cloud. Recently, tests were integrated to execute during the build phase, preventing any version disruptions in AWS. Additionally, maintainers can now trigger tests in Pull Requests, streamlining the validation of contributions against AWS. This advancement significantly enhances Wing's testing efficiency and reliability. 
+
+
+### Upcoming Events
+
+**🔥 Live Workshop - December 6th @4:30 - 6pm IST**
+Get hands-on with Eyal as he walks us through the process of constructing a full-fledged React.js application with Winglang.
+For more information and to register, [click here](https://www.eventbrite.com/e/winglang-react-workshop-tickets-754616256537?aff=oddtdtcreator).
+
+### The Wingly Update
+Have you heard about **[The Wingly Update](https://www.twitch.tv/winglangio)**? It's our biweekly show where we stream to Twitch, share the latest developments of the project, and just make a bunch of fools of ourselves. Mark your calendars and make sure to catch us live!
+
+If you haven't been able to catch our show, don't fret! You can find the complete stack of [all our episodes here](https://youtube.com/playlist?list=PL-P8v-FRassZBWsNoSafL_ReO0JO0xJVm&si=trffVrtGGMUZ-SKb). 
+
+You can check out some cool highlights from our show below!
```

Comment:
let's add more segments here. maybe ~10? maybe just focus on interviews?
also - i wouldn't put the changelog segments, just interviews.